### PR TITLE
Fix some of the CLI commands and fix logic with the mod installer fallback to advanced installer

### PR DESCRIPTION
**Pull request by halgari**
Fixes a regression caused by incorrect boolean logic around how we fallback to advanced installers. It's a 3-state situation, not a 2 state

### CODE REVIEW

File: src/NexusMods.Library/InstallLoadoutItemJobWorker.cs [hunk 1]
```diff
@@ -34,8 +34,11 @@ protected override async Task<JobResult> ExecuteAsync(InstallLoadoutItemJob job,
 
         var result = await ExecuteInstallersAsync(job, installers, cancellationToken);
 
-        if (!result.HasValue && job.Installer is not AdvancedManualInstaller)
+        if (!result.HasValue)
         {
+            if (job.Installer is not AdvancedManualInstaller)
+                return JobResult.CreateFailed($"Advanced installer did not succeed for `{job.LibraryItem.Name}` (`{job.LibraryItem.Id}`)");
```
Comment (erri120) [inline src/NexusMods.Library/InstallLoadoutItemJobWorker.cs:40]:
```suggestion
            if (job.Installer is AdvancedManualInstaller)
                return JobResult.CreateFailed($"Advanced installer did not succeed for `{job.LibraryItem.Name}` (`{job.LibraryItem.Id}`)");
```

**Review by erri120 — CHANGES_REQUESTED** [review]

File: src/NexusMods.Library/InstallLoadoutItemJobWorker.cs (hunk 1 above)
Comment (erri120) [reply src/NexusMods.Library/InstallLoadoutItemJobWorker.cs:40]: If the provided installer was the advanced installer, then we don't want to run it again if it failed/user cancelled.

Comment (Al12rs) [inline src/NexusMods.Library/InstallLoadoutItemJobWorker.cs:40]: This doesn't sound right though, if the last installer is _not_ the Advanced one and the result is empty we should attempt using the Advanced installer.

**Review by Al12rs — CHANGES_REQUESTED** [review]

File: src/NexusMods.Library/InstallLoadoutItemJobWorker.cs (hunk 1 above)
Comment (Al12rs) [reply src/NexusMods.Library/InstallLoadoutItemJobWorker.cs:40]: erri beat me to it

**Pull request merged**